### ISSUE TITLE
Hack understand

### ISSUE THREAD

**Issue by sowe** [jortilles/byPassPentahoLogin#1]
Hi Juanjo,
I'm try to use this hack. But i don't understand  to how to use the hack. 
I've this in catalina.out:
`llamada con token: fccef24e1e7ee5f999af67ce0783cdab para el destino: destination_1
Digest(in hex format):: fccef24e1e7ee5f999af67ce0783cdab`
and this is my 
destinos.properties in **../biserver-ce/tomcat/webapps/pentaho/WEB-INF/classes**

> userid=admin
> password=password
> dst1=/pentaho/api/repos/%3Apublic%3ASteel%20Wheels%3ADashboards%3ACTools_dashboard.wcdf/generatedContent
> ![screen shot 2016-04-06 at 10 25 29](https://cloud.githubusercontent.com/assets/482053/14310402/ec1a52a0-fbe1-11e5-9239-8a4eccb73321.png)

**Comment by jortilles** [comment]
as you defined a destination as dst1 you shoul make the call as http://localhost:8080/pentaho/Urbano?token=e99b5cdc07594f1e7bed336bbaf2e2db&dst=dst1

I've just checked and it works fine for pentaho 6.0.1

**Comment by sowe** [comment]
OK i'll try and how generate a new jar .... ?

**Comment by sowe** [comment]
Hi @jortilles 
I need your help, i'm try to install your hack  but I always ask user and password. I'm flowing:
- biserver-ce/tomcat/webapps/pentaho/WEB-INF/classes/destinos.properties
  [destinos.txt](https://github.com/jortilles/byPassPentahoLogin/files/253428/destinos.txt)
- biserver-ce/tomcat/webapps/pentaho/WEB-INF/web.xml (in this case web.txt)
  [web.txt](https://github.com/jortilles/byPassPentahoLogin/files/253427/web.txt)
  Any idea?

**Comment by jortilles** [comment]
Let's do a hangouts....

Wednesday?

Juanjo Ortilles - Open Business Analytics
twitter: jortilles
www.jortilles.com / es / cat

2016-05-07 15:10 GMT+02:00 Rafael notifications@github.com:

**Comment by DesarrolloSGSOFT** [comment]
i've the same problem, how you can fixed?

**Comment by jortilles** [comment]
Sowe did something related recently also...

https://github.com/sowe/byPassPentahoLogin

Juanjo Ortilles - Open Business Analytics
twitter: jortilles
www.jortilles.com / es / cat

2016-07-06 0:38 GMT+02:00 DesarrolloSGSOFT notifications@github.com:

**Comment by DesarrolloSGSOFT** [comment]
Hi Again, how can i change token??
http://localhost:8080/pentaho/Urbano?token=e99b5cdc07594f1e7bed336bbaf2e2db&dst=destination_1

specifically:  e99b5cdc07594f1e7bed336bbaf2e2db,
where i must to do changues??..etc....

Thanks...

**Comment by jortilles** [comment]
Everithing is in the documentation....

Juanjo Ortilles - Open Business Analytics
twitter: jortilles
www.jortilles.com / es / cat

2016-07-06 19:47 GMT+02:00 DesarrolloSGSOFT notifications@github.com:

**Comment by jortilles** [comment]
In spanish:
http://pentahohispano.blogspot.com.es/2015/10/bypass-pentaho-login-acceso-desde.html

Juanjo Ortilles - Open Business Analytics
twitter: jortilles
www.jortilles.com / es / cat

2016-07-06 19:47 GMT+02:00 DesarrolloSGSOFT notifications@github.com:

**Comment by DesarrolloSGSOFT** [comment]
Yes, I read spanish page, but , don't show me how to do it, also read documentation, but i have the same problem...
